src/components/results/ListItem.vue

```diff
@@ -0,0 +1,37 @@
+<script setup>
+import {ref} from 'vue';
+
+const props = defineProps({
+  item: Object,
+});
+const obj = ref(props.item);
```

Comment:
you don't need to wrap it in ref, you can use all your props in themplates without it